source/index.html.md

```diff
@@ -333,64 +333,71 @@ Depending on card scheme and merchant category, the name on the card might be
 necessary for approval of credits. It may be provided through the optional
 parameter `card[name]`.
 
-## Recurring payments
+## Series of transactions
 
-Recurring payments enable you to repeatedly charge cardholders without providing
-CSC or other means of cardholder authentication for subsequent payments.
+Clearhaus currently support only one type of subscription billing:
 
-### Subscription concept
+* Recurring: Transactions processed at predetermined, regular intervals not
+    exceeding 1 year; e.g. a monthly subscription for a magazine.
```

Comment:
Perhaps make it explicitly clear if a 12 month interval is ok or not.

Reply:
🤔 I am not even sure we know for sure; WDYT @AlexanderIversen?

Reply:
From Mastercard Transaction Processing Rules I have the following quote: 

> Use may occur periodically, such as on a monthly, quarterly, or
annual basis, or as needed to “top up” the Cardholder’s account with the
Merchant (Transaction Processing Rules, 13 August 2020, p. 158)

This is not very specific but I agree that intervals should not exceed 1 year (1 year and 1 day) but I agree with @mt-clearhaus that an interval can be 1 year is OK. 

From Visa Core Rules I have the following: 

> A Transaction in a series of Transactions that use a Stored Credential
and that are processed at fixed, regular intervals (not to exceed one
year between Transactions), representing Cardholder agreement for
the Merchant to initiate future Transactions for the purchase of goods
or services provided at regular intervals. (Visa Core Rules, 18 August 2020, p. 1126)

Visas statement is actually more precise. They state that intervals cannot exceed 1 year between transactions.

Reply:
Leap years? 😆

Reply:
```suggestion
    exceeding 366 days; e.g. a monthly subscription for a magazine.
```
WDYT?

Reply:
I like it :smile: :+1: Because of `>` rather than `>=` it allows for leap years.

Reply:
But then, uhm, permaybehaps "exceeding 1 year" isn't too bad after all :thinking: 

When reading it the first time, I wasn't sure it would be ok to charge once a year, but the second time around I find it "not that bad".

Reply:
I think I'll keep the original text and just allow it to be imprecise. If necessary we'll elaborate in the guideline or fix if enough people are asking 🙂